pkg/cluster/spec/pd.go

```diff
@@ -52,30 +51,19 @@ type PDSpec struct {
 }
 
 // Status queries current status of the instance
-func (s PDSpec) Status(tlsCfg *tls.Config, pdList ...string) string {
-	curAddr := fmt.Sprintf("%s:%d", s.Host, s.ClientPort)
-	curPdAPI := api.NewPDClient([]string{curAddr}, statusQueryTimeout, tlsCfg)
-	allPdAPI := api.NewPDClient(pdList, statusQueryTimeout, tlsCfg)
+func (s PDSpec) Status(tlsCfg *tls.Config, _ ...string) string {
+	addr := fmt.Sprintf("%s:%d", s.Host, s.ClientPort)
+	pc := api.NewPDClient([]string{addr}, statusQueryTimeout, tlsCfg)
 	suffix := ""
```

Comment:
The suffix will always be empty since you delete the `find dashbaord` stage. So I think we can delete it this time

Reply:
PTAL